pyk/src/pyk/konvert/_module_to_kore.py

```diff
@@ -1063,6 +1196,7 @@ class DiscardHookAtts(SingleModulePass):
         'SET',
         'STRING',
         'SUBSTITUTION',
+        'TIMER',
         'UNIFICATION',
     )
 
```

Comment:
At ~1300 lines for this file, maybe worth refactoring into separate modules...
What comes to mind is a split into compiler passes and final axiom rendering.